src/components/ui/ShippingSimulation/useShippingSimulation.ts

```diff
@@ -134,8 +109,28 @@ const reducer = (state: State, action: Action) => {
   }
 }
 
-export const useShippingSimulation = () => {
-  const { postalCode: sessionPostalCode } = useSession()
+function getShippingInformation(
+  shipping: ShippingSimulationQueryQuery['shipping']
+): [string, ShippingOptionProps[]] {
+  const location =
+    [shipping?.address?.neighborhood, shipping?.address?.city]
+      .filter(Boolean)
+      .join(' / ') ?? ''
+
+  const options: ShippingOptionProps[] =
+    shipping?.logisticsInfo?.[0]?.slas?.map((sla) => ({
+      carrier: sla?.name ?? '',
```

Comment:
What do you think of favoring `friendlyName`? I haven't been able to query something that would return this field, but I imagine it's a more human-friendly version of `name`, when it's available.

<img width="361" alt="CleanShot 2022-08-25 at 17 26 39@2x" src="https://user-images.githubusercontent.com/381395/186762666-119495bb-c5cb-46de-8022-fd5d28a9b6d0.png">

```suggestion
      carrier: sla?.friendlyName ?? sla?.name ?? '',
```